src/io/iter_image_det_recordio.cc

```diff
@@ -237,53 +237,53 @@ class ImageDetRecordIOParser {
   std::unique_ptr<ImageDetLabelMap> label_map_;
   /*! \brief temp space */
   mshadow::TensorContainer<cpu, 3> img_;
+  /*! \brief OMPException obj to store and rethrow exceptions from omp blocks*/
+  dmlc::OMPException omp_exc_;
 };
 
-template<typename DType>
+template <typename DType>
 inline void ImageDetRecordIOParser<DType>::Init(
-    const std::vector<std::pair<std::string, std::string> >& kwargs) {
+    const std::vector<std::pair<std::string, std::string>> &kwargs) {
```

Comment:
The space is left deliberately for compatibility with some compilers, don't remove it.

Reply:
I have added it back. Which compiler is it ? Maybe it should be added to the CI.